sophiread/environment_mac.yml

```diff
@@ -13,6 +13,10 @@ dependencies:
   - eigen
   - tbb-devel
   - spdlog
+  #- mlpack
+  - cereal
+  - armadillo
+  - ensmallen
```

Comment:
if only enables `FastSophiread`, there is no need to include anything that related to `mlpack` as we are planning to switch to a native implementation of DBSCAN instead of using one provided from `mlpack`.

> the performance of `mlpack` is way below what we need, and it does not have native m-series support (not in their timeline either). So we decided to drop `mlpack` as dependency in favor of a native implementation (in the work).